src/pages/students/forms/paymentDataForm/PaymentData.tsx

```diff
@@ -1,77 +1,69 @@
 import { MainContainer, 
-        TitleContainer, 
         FormContainer, 
         Label, 
         Input, 
-        Title, 
         Button, 
         ButtonsContainer, 
         Select, 
         SmallInput, 
         InputsContainer,
-        Text} from "../styles/PaymentData.styles";
-import { PaymentPlanName } from "../../../app/types/models/PaymentPlanName";
+        Text} from "./PaymentData.styles";
+import { PaymentPlanName } from "../../../../app/types/models/PaymentPlanName";
 import { useForm } from "react-hook-form";
 import { yupResolver } from "@hookform/resolvers/yup";
 import * as yup from "yup";
-import { paymentDataScheme } from "../schemes/PaymentData.scheme";
-import { ErrorMessage } from "../../../components/header/ErrorMessage";
-import { useNavigate } from "react-router";
-import { setStudentData } from "../StudentRegistrationFormSlice";
-import type { RootState } from "../../../app/store/store";
-import { useDispatch, useSelector } from "react-redux";
+import { paymentDataScheme } from "./PaymentData.scheme";
+import { ErrorMessage} from "../../../../components/error_message/ErrorMessage"
+import type { PaymentDataProps } from "../../create-student/CreateStudent";
 
+interface FormProp {
+  createStudentCall: (data: PaymentDataProps) => void;
+  paymentData:PaymentDataProps | undefined;
+  stepBack:()=>void;
+}
 
-function PaymentData() {
+function PaymentData({
+  createStudentCall,
+  paymentData,
+  stepBack
+}: FormProp) {
```

Comment:
De modo que aca, podriamos hacer lo siguiente


```suggestion
function PaymentData({
  onNext,
  onBack,
  data
}: FormProp<PaymentDataProps>) {
```

Fijate que en la misma definicion del formulario, estamos definiendo el tipo de data que vamos a recibir segun la interfaz que usamos `FormProp<PaymentDataProps>`, esto es gracais al generico `T` que describimos en la interfaz